scripts/refresh-after-snapshot.py

```
@@ -428,28 +428,78 @@ def pre_fix_cert_sans(config: RefreshConfig) -> None:
+    has_errors = oc(
+        "get", "keycloakrealmimport", "osac-realm-import", "-n", config.keycloak_ns,
+        "-o", "jsonpath={.status.conditions[?(@.type==\"HasErrors\")].status}",
+        capture=True, check=False,
+    ).stdout.strip()
+    if has_errors == "True":
+        msg = oc(
+            "get", "keycloakrealmimport", "osac-realm-import", "-n", config.keycloak_ns,
+            "-o", "jsonpath={.status.conditions[?(@.type==\"HasErrors\")].message}",
+            capture=True, check=False,
+        ).stdout.strip()
+        raise RuntimeError(f"KeycloakRealmImport failed: {msg}")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)